packages/react/src/select-value/SelectValue.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { useSelectContext } from "../select-context";
+
+type SelectValueProps = BoxProps<"div", { placeholder?: string }>;
+
+export const SelectValue = forwardRef<HTMLDivElement, SelectValueProps>(
+  ({ children, placeholder, ...props }, ref) => {
+    const { downshift, itemToString } = useSelectContext("SelectValue");
+    return (
+      <Box {...props} ref={ref}>
+        {(children ?? downshift.selectedItem)
+          ? itemToString(downshift.selectedItem)
+          : placeholder}
```

Comment:
this doesn't look right - it should be something like:
```suggestion
        {children ??
          (downshift.selectedItem
            ? itemToString(downshift.selectedItem)
            : placeholder)}
```